docs/configuration/logs.md

```diff
@@ -124,6 +124,13 @@ filePath = "/path/to/access.log"
 format = "json"
 ```
 
+To write the logs in async, specify `bufferingSize` as the format (must be >0):
```

Comment:
If I understand right the code, the buffer size is about the number of lines to bufferize before to write them into the log file.
As usualy the buffer size can concern a number of bytes, I guess it should be nice to precise in the documentation that the buffer size is about a number of access logs and not a number of bytes.

WDYT?